### DISCUSSION TITLE
Möglichkeit zur Konfiguration

### DISCUSSION THREAD

**Discussion by Philip-Wiege** [MyHomeMyData/ha.vis.vitocal250#4] [Ideas]
Hallo zusammen,

lassen sich in HA eigentlich Konfigurationsvariablen definieren? Zum Beispiel Heizungspuffer ja/nein oder Anzahl Ventilatoren Außeneinheit?

Eventuell lassen sich so ja Teile des GUI ausblenden. Ich fahre zum Beispiel ohne Heizungspuffer 😉

Grüße
Philip

**Comment by jwende** [comment]
So etwa? https://www.home-assistant.io/docs/scripts/#scope-of-variables

**Comment by aleex42** [comment]
Ich glaube das wird schwierig das mit der Picture Elements Card zu vereinen.

Notfalls kannst du ja aber über die state filter das CSS anpassen. Aber dann kannst du die nicht vorhandenen Zeilen auch per Hand rauswerfen ... :-(

**Comment by Philip-Wiege** [comment]
Mal sehen was so geht...
Momentan versuche ich das App Interface nachzubauen.

![image](https://github.com/MyHomeMyData/ha.vis.vitocal250/assets/19406918/8d81ad89-13bb-4e62-918d-91f7a60b8444)
